crane_x7_moveit_config/config/joint_limits.yaml

```diff
@@ -2,52 +2,52 @@
 # Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
 # Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
 
-default_velocity_scaling_factor: 1.0
-default_acceleration_scaling_factor: 1.0
+default_velocity_scaling_factor: 0.5
```

Comment:
what if we don't do scaling but keeping the 1/10th reductions below?

Reply:
it would be interesting to log the joint velocities and compare before/after changes